reddit.js

```diff
@@ -1,5 +1,52 @@
 var reddit = require("./data.json")
 
+
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+/*Part 1 - forEach - console.log all titles*/
+
+reddit.data.children.forEach(function (theTitle){
```

Comment:
Here, I would recommend ```child``` as a more appropriate parameter than ```theTitle```, since ```forEach``` is iterating over each child of the ```reddit.data.children``` array.